pgmpy/tests/test_causal_discovery/test_PC.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from sklearn.utils.estimator_checks import check_estimator
+
+from pgmpy.causal_discovery import PC
+
+
+class TestPCEstimatorClass(unittest.TestCase):
+    def setUp(self):
+        return super().setUp()
+
+    def test_sklearn_compatibiltity(self):
+        pc_estimator = PC()
+
+        try:
+            check_estimator(pc_estimator)
+        except Exception as e:
+            self.fail(f"PC estimator is not an sklearn-type Estimator: {e}")

```

Comment:
Could you please also replace this test with something like:
```python
def make_estimator():
    return PC()


@parametrize_with_checks([make_estimator()])
def test_pc_compatibility(estimator, check):
    check(estimator)
```
I think it runs the same checks but shows a better logging message, which might be helpful for debugging failures.